.prettierrc.js

```diff
@@ -4,8 +4,6 @@ module.exports = {
   printWidth: 80,
   singleQuote: false,
   trailingComma: "es5",
-  jsxBracketSameLine: false,
```

Comment:
modern prettier was complaining this is a deprecated rule, and i removed the jsx stuff from linting anyway so it's not really relevant